vpr/src/route/rr_graph_indexed_data.cpp

```diff
@@ -115,6 +115,18 @@ void alloc_and_load_rr_indexed_data(const std::vector<t_segment_inf>& segment_in
     load_rr_indexed_data_T_values((CHANX_COST_INDEX_START + num_segment),
                                   num_segment, CHANY, nodes_per_chan, L_rr_node_indices);
 
+    for (int cost_index = CHANX_COST_INDEX_START;
```

Comment:
Since this fix-up loop is really setting the T_values I think it should be inside `load_rr_indexed_data_T_values()`. 

It would also be good to further expand on the comment to describe more conceptual context, including:
 * When this case could occur (e.g. segment type only in one axis)
 * What we are assuming by re-using the ortho direction T values

Reply:
> Since this fix-up loop is really setting the T_values I think it should be inside `load_rr_indexed_data_T_values()`.

It doesn't belong in `load_rr_indexed_data_T_values()` because it requires that the other side be done already.  `load_rr_indexed_data_T_values()` is called twice, once for `CHANX` and once for `CHANY`.  During the `CHANX` call, the orthogonal data won't have been loaded yet.

Reply:
I've expanded the comment, hopefully it addresses your concerns.

Reply:
> It doesn't belong in `load_rr_indexed_data_T_values()` because it requires that the other side be done already. `load_rr_indexed_data_T_values()` is called twice, once for `CHANX` and once for `CHANY`. During the `CHANX` call, the orthogonal data won't have been loaded yet.

OK makes sense, in that case its probably best to put it into a named function.

Reply:
Done